### ISSUE TITLE
Unit tests failing due to externalGenerator files missing

### ISSUE THREAD

**Issue by wdconinc** [JeffersonLab/remoll#100]
```
G4WT1 > Setting the external file to ../analysis/externalGenerator/remollGenFunctions.root
Error in <TFile::TFile>: file ../analysis/externalGenerator/remollGenFunctions.root does not exist
G4WT1 > could not find function in file ../analysis/externalGenerator/remollGenFunctions.root
```

**Comment by wdconinc** [comment]
Like, all tests are failing...

**Comment by wdconinc** [comment]
https://travis-ci.org/JeffersonLab/remoll/builds/391450918

**Comment by wdconinc** [comment]
Fixed by 59e4010cf656cf3bdef56f861b197e9c23621744
